pymc/distributions/distribution.py

```diff
@@ -61,7 +62,7 @@
     "NoDistribution",
 ]
 
-DIST_PARAMETER_TYPES = Union[np.ndarray, int, float, TensorVariable]
+DIST_PARAMETER_TYPES: TypeAlias = Union[np.ndarray, int, float, TensorVariable]
```

Comment:
Im just curiouys, what does TypeAlias do?

Reply:
This is a type hint itself. The mypy errors and documentation told me that aliases should be annotated like this.